__tests__/index.test.js

```diff
@@ -108,6 +108,38 @@ describe('/api/articles', () => {
         });
       });
   });
+  test('GET:200 Responds with specific articles (articles array) of article objects when topic is specified', () => {
+    const topic = "mitch"
+    return request(app)
+      .get('/api/articles')
+      .query({topic})
+      .expect(200)
+      .then((response) => {
+          expect(response.body.articles.length).toBe(12);
+        response.body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url : expect.any(String),
+            body: expect.any(String)
+          })
```

Comment:
We should be asserting that every topic is mitch here otherwise there is no way of actually knowing if the filter worked.

Reply:
Added this check. This change will be part of re-commit of  Task 11.